sunpy/lightcurve/sources/soho.py

```diff
@@ -0,0 +1,207 @@
+# -*- coding: utf-8 -*-
+"""SOHO LightCurve sources subclass definitions"""
+
+from __future__ import absolute_import
+
+
+__authors__ = ["Ankit Kumar"]
+__email__   = "ankitkmr.iitk@gmail.com"
+
+# This module was developed with funding from 
+# Google Summer of Code 2015
+
+
+from datetime import timedelta,datetime,time
+
+import pandas as pd
+from pandas import DataFrame
+import numpy as np
+from matplotlib import pyplot as plt
+import matplotlib.dates
+
+from astropy.io import ascii
+from astropy.table import Table, Column
+from astropy.utils import OrderedDict
+from astropy.table.column import MaskedColumn
+
+from sunpy.time import TimeRange
+from sunpy.lightcurve import LightCurve
+
+
+
+__all__ = ['ERNELightCurve']
+
+
+#This code for _to_pandas is repeated here and in stereo.py for performance reasons.
+def _to_pandas(self):
+	"""
+	Return a :class:`pandas.DataFrame` instance
+
+	Returns
+	-------
+	dataframe : :class:`pandas.DataFrame`
+		A pandas :class:`pandas.DataFrame` instance
```

Comment:
please, do not use `tabs` to indent, but `spaces`. Check the [1st point](http://astropy.readthedocs.org/en/latest/development/codeguide.html#coding-style-conventions) on the `astropy` guidelines.